src/frida/repl.py

```diff
@@ -78,15 +89,37 @@ def _unload_script(self):
 
         def _process_input(self, reactor):
             self._print_startup_message()
-            self._ready.wait()
+            while self._ready.wait(0.5) != True:
+                if not reactor._running:
```

Comment:
You are right, but this is about writing idiomatic code and avoid confusing people reading the code who will be wondering "why is this private variable accessed from outside the class, and without taking the lock?".